bftengine/include/bftengine/DbCheckpointManager.hpp

```diff
@@ -33,6 +33,7 @@
 #include "InternalBFTClient.hpp"
 #include "storage/db_interface.h"
 #include "util/filesystem.hpp"
+#include "kv_types.hpp"
```

Comment:
bft engine shouldn't include kvbc as it creates a cyclic dependency

Reply:
thanks for the catch @toly-kournik ! will change it retroactively after the merge